front/src/modules/trainschedule/components/Timetable/PacedTrain/OccurrenceItem.tsx

```diff
@@ -110,6 +121,7 @@ const OccurrenceItem = ({
       title: t('occurrenceMenu.delete'),
       icon: <Trash />,
       onClick: () => {
+        deleteAddedException(occurrenceId);
```

Comment:
very nit: what about occurrence.id ?
```suggestion
        deleteAddedException(occurrence.id);
```

Reply:
yes that’s where it comes from
```
  const {
    id: occurrenceId,
    trainName,
    rollingStock,
    startTime,
    disabled,
    exceptionChangeGroups,
  } = occurrence;
```

Reply:
I'm wondering if it's really useful to extract it from the occurrence. Maybe it's simpler to directly call it from the occurrence

Reply:
Anyway, it's nit, you can close this conversation

Reply:
ah yes I see, made the change